### PR TITLE
Remove elm-css and NoRedInk/datetimepicker-legacy dependencies

### PR DESCRIPTION
**Pull request by tesk9**
`NoRedInk/datetimepicker-legacy` is unmaintained (and not 100% functional), and it's unlikely to be updated to the `elm-css` 18.0.0. `elm-debug-controls` depended on `elm-css` only because it was depending on `datetimepicker-legacy`. By replacing `datetimepicker-legacy` with an HTML input with [type "datetime-local"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local), it's possible to remove `datetimepicker-legacy` and `elm-css` dependencies.

Hopefully, this will make package maintenance a little bit less tedious -- few dependencies means fewer updates because somebody else changed something.

## Before

Screen Shot showing custom datetimepicker with an analog timepicker

## After

Screen Shot showing default datetimepicker

### CODE REVIEW

File: src/Debug/Control.elm [hunk 1]
```diff
@@ -201,60 +199,173 @@ stringTextarea initialValue =
...
+                ++ "-"
+                ++ twoDigit initial.day
+                ++ "T"
+                ++ twoDigit initial.hour
+                ++ ":"
+                ++ twoDigit initial.minute
     in
-    date_ (DateTimePicker.initialStateWithToday initialDateTime) initialDateTime
-        |> map toPosix
+    date_ initialDateTime
```
Comment (avh4) [inline src/Debug/Control.elm:222]: :art: Now that `date_` is clearly meant to deal with only the `String` representation, I think maybe `dateField` or `dateInputField` would be a good name for it?

**Review by avh4 — APPROVED** [review]
Looks good!  I just have to remember how to run this locally again...

**Comment by avh4** [comment]
I'm gonna address the comments.

**Comment by avh4** [comment]
https://package.elm-lang.org/packages/avh4/elm-debug-controls/2.2.3/

**Comment by avh4** [comment]
Such a relief to remove the dependency!!

**Comment by tesk9** [comment]
Yay! 🎉 Thank you!

**Pull request merged**